frontend/src/models/models.js

```diff
@@ -67,6 +68,16 @@ module.exports = app => app.component('models', {
       
       await this.getDocuments();
     },
+    isSubdoc(schemaPath) {
+      console.log('what is schemaPath in isSubdoc', schemaPath);
+      if (typeof schemaPath.options?.type?.tree === 'undefined') {
```

Comment:
So the issue here is that, for subdocuments, `schemaPath` is "corrupted". The issue comes down to this commit in Mongoose: https://github.com/Automattic/mongoose/commit/6399f3ca9084776a3d9688811e22f4a301452bc5 , which I will likely try to undo.

But I think the correct fix for this issue is to make sure that we always have `instance` for a schema path in `getDocuments()`. Something like `schemaPaths = {}; for (const path of Object.keys(Model.schema.paths)) { schemaPaths[path] = { instance: Model.schema.paths[path] } }`

Reply:
I made your change but not exactly sure how this helps when you patch the issue. The way you have the frontend set up I'm  now having to add an extra`.instance` to where `schemaPath` is accessed or passed but doesn't seem to resolve the subdoc issue.